megalodon/src/notification.ts

```diff
@@ -5,6 +5,7 @@ namespace NotificationType {
   export const Favourite: Entity.NotificationType = 'favourite'
   export const Reblog: Entity.NotificationType = 'reblog'
   export const Mention: Entity.NotificationType = 'mention'
+  export const Reaction: Entity.NotificationType = 'reaction'
```

Comment:
Why did you add a new notification type? Is it used?

Reply:
Yes, here: https://github.com/h3poteto/megalodon/pull/1921/files#diff-727a65323e887a8055dfc36e06244e7622dc446801b15f1027995e8b4c7cf63dR367

Reply:
Why did you add Reaction even though EmojiReaction exists? What's the difference?

Reply:
Mostly to stay in line with Firefish's `reaction` notification name. I don't think it's very necessary but helps with differentiating string-only reaction notifications (Pleroma) from Reaction object notifications

Reply:
Ah, I got it.
You mean
In Pleroma(current):
When we receive an EmojiReaction notification, it contains `emoji` field in the object (it does not contain `reaction`). And it should be `NotificationType.EmojiReaction`.
In Firefish:
When we receive a Reaction notification, it contains `reaction` field in the object (it does not contain `emoji`). And it should be `NotificationType.Reaction`.

Furthermore, in the future Pleroma, when we receive an EmojiReaction, it contains `reaction` field in the object instead of `emoji` field. And at that time, we can remove `NotificationType.EmojiReaction`.

Correct?

Reply:
Yep, you got it.